.eslintrc.yml

```diff
@@ -13,6 +13,7 @@ parserOptions:
   sourceType: module
 rules:
   class-methods-use-this: off
+  max-len: [1, 120]
```

Comment:
@RomanHotsiy , how is max-len used? Could not find it anywhere else in the code. Also, if it's for identifiers' lengths should not it be an integer instead of an array? (which I suppose indicates MIN and MAX lengths)

Reply:
It's .eslintrc, it's for max code line length in our code (the default from airbnb-base is 100 which I find quite outdated for modern screens)

Reply:
Ouch, missed the filename :(